sys-auth/nss-mdns/nss-mdns-0.15.1.ebuild

```diff
@@ -18,6 +18,15 @@ RDEPEND=">=net-dns/avahi-0.6.31-r2[${MULTILIB_USEDEP}]"
 DEPEND="${RDEPEND}
 	test? ( >=dev-libs/check-0.11[${MULTILIB_USEDEP}] )"
 
+PATCHES=(
+	"${FILESDIR}"/lld-17-undefined-versioned-symbols.patch
+)
+
+src_prepare() {
+	default
+	eautoreconf
```

Comment:
I'll add a comment that this is only there for the patch, so we can drop it in future.